### PR TITLE
Update tr.json

### PR DESCRIPTION
**Pull request by altayaydemir**
## Description
Adds missing translate strings for Turkish.

## Requirements Checklist
- [ ] Feature implemented / Bug fixed
- [ ] If necessary, more likely in a feature request than a bug fix
  - [ ] Change has been verified in an actual browser (Chome, Firefox, IE)
  - [ ] Unit Tests updated or fixed
  - [ ] Docs/guides updated
  - [ ] Example created ([starter template on JSBin](http://jsbin.com/axedog/edit?html,output))
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

File: lang/tr.json [hunk 1]
```diff
@@ -15,20 +16,62 @@
   "Playback Rate": "Oynatma Hızı",
   "Subtitles": "Altyazı",
   "subtitles off": "Altyazı Kapalı",
-  "Captions": "Ek Açıklamalar",
-  "captions off": "Ek Açıklamalar Kapalı",
+  "Captions": "Altyazı",
```
Comment (gkatsev) [inline lang/tr.json]: I guess there's not much difference between subtitles and captions in turkish?

Comment (altayaydemir) [inline lang/tr.json]: Exactly.

**Comment by altayaydemir** [comment]
Thanks @gkatsev 👍

**Comment by gkatsev** [comment]
Thanks @altayaydemir!

**Pull request merged**